### PR TITLE
Dijkstras algo

### PR DESCRIPTION
**Pull request by osherboudara99**

### CODE REVIEW

**Comment by eytan-ohana** [comment]
looks good add markdown make it pretty 
@osherboudara99

**Comment by eytan-ohana** [comment]
In the notebook specify you use the adjacency matrix
In the readme Give an intro to adjacency matrices in the right place

File: Dijkstras-Algo.ipynb [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    "G.add_edge(3, 4,weight=4.0)\n",
+    "G.add_edge(1, 5,weight=6.0)\n",
+    "G.add_edge(3, 6,weight=7.0)\n",
+    "G.add_edge(4, 6,weight=2.0)\n",
+    "G.add_edge(5, 6,weight=2.0)\n",
+    "draw_graph(G)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (eytan-ohana) [inline Dijkstras-Algo.ipynb:388]: add comment/markdown about adj matrix in notebook

**Review by eytan-ohana — CHANGES_REQUESTED** [review]

File: Dijkstras-Algo.ipynb (hunk 1 above)
Comment (osherboudara99) [reply Dijkstras-Algo.ipynb:388]: on it

**Comment by eytanohana** [comment]
@osherboudara99 The notebook looks great I like it a lot! A couple things:
* Can we clean up the pseudo-code in the notebook right now its a lot to look at.
* in the algorithm i dont see a need to pass in the graph as well as the adjacency matrix cause they both represent the same thing. you just use G to get the number of nodes you can replace that with the number of rows/columns in A.
* I really like that it works out of the box on directed/undirected/weighted/unweighted graphs. 
* I think its time we readjust the readme again and seriously talk about adjency lists vs matrices.

**Pull request merged**